types/react/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for React 15.6
+// Type definitions for React 16.0
```

Comment:
I don't know how dtslint works here but I assume it's expecting you to copy the react 15 `index.d.ts` to `react/v15/`.